bin/_test-helpers.sh

```diff
@@ -381,6 +381,18 @@ upgrade_test() {
   fi
 
   install_version "$install_url" "$upgrade_version"
+
+  # Install Viz if the version is edge
+  # TODO: Enable this also for stable once 2.10 is out
+  if [ "$release_channel" == "edge" ]; then
```

Comment:
Better use the POSIX compatible `=` instead of `==` (they are equivalent in bash).
Also, no need for double quotes for static strings, use single quotes or no quotes.

`if [ "$release_channel" = 'edge' ]; then`